kahuna/public/js/search/index.js

```diff
@@ -149,7 +149,19 @@ search.config(['$stateProvider', '$urlMatcherFactoryProvider',
 
                     const sub = dragData$.subscribe(({ images, dt }) => {
                         if (images.size > 0) {
+                            //creates an element to use as the drag icon
+                            const dragImage = document.createElement('div');
+                                  dragImage.classList.add("dragIcon");
+
+                            const imageCount = document.createElement('span');
+                                  imageCount.classList.add("dragCount");
+
+                                  imageCount.innerHTML = images.count();
+                            dragImage.appendChild(imageCount);
+                            document.body.appendChild(dragImage);
```

Comment:
Looking at the example [here](https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/setDragImage) it doesn't have to be, but I am sure I have seen it somewhere that it might.